O/oxigraph_server/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "oxigraph_server"
+version = v"0.2.5"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/oxigraph/oxigraph.git", "a21dcbb4f7355d7a00a86fbc5ad2c350a53629c4"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/oxigraph/server
+
+export OPENSSL_STATIC=yes
```

Comment:
Why static?  If that's static, OpenSSL is probably not a runtime dependency then?

Reply:
It doesn't build if not static, so I'll change it to a build dependency?

Reply:
Actually the program does link to OpenSSL:
```julia
shell> $(libtree()) oxigraph_server
oxigraph_server 
├── libssl.so.1.1 [default path]
│   └── libcrypto.so.1.1 [default path]
├── librt.so.1 [default path]
└── libcrypto.so.1.1 [default path]

julia> using libtree_jll

shell> $(libtree()) oxigraph_server
oxigraph_server 
├── libssl.so.1.1 [default path]
│   └── libcrypto.so.1.1 [default path]
├── librt.so.1 [default path]
└── libcrypto.so.1.1 [default path]
```
So I'm not sure what `OPENSSL_STATIC=yes` means.

Reply:
Where did you find [`OPENSSL_STATIC`](https://github.com/oxigraph/oxigraph/search?q=OPENSSL_STATIC)?

Reply:
https://github.com/JuliaPackaging/Yggdrasil/blob/4dc836674457ca90a72830e4d4a4f80aa27278db/0_RootFS/Rust/build_tarballs.jl#L65

Reply:
That doesn't explain what's the purpose of that line

Reply:
Fair point.

Reply:
Turns out, it is utterly unnecessary.